docs/topics/http/file-uploads.txt

```diff
@@ -38,9 +38,9 @@ subclass) in the form. So the data from the above form would
 be accessible as ``request.FILES['file']``.
 
 Note that :attr:`request.FILES <django.http.HttpRequest.FILES>` will only
-contain data if the request method was ``POST`` and the ``<form>`` that posted
-the request has the attribute ``enctype="multipart/form-data"``. Otherwise,
-``request.FILES`` will be empty.
+contain data from form fields if the request method was ``POST``, at least one
+file field was actually posted, and the ``<form>`` that posted the request has
```

Comment:
I notice Oxford commas used quite a bit in the docs. Is there a reason for this? Personally, I'm not sure they aid readability in scenarios such as this. 

An example where I think they are helpful: 'I went shopping at marks and spencers, and tesco'. 

I guess what I'm really asking is this a conscious 'house style'? If so I'll bear in mind for the future. :-)

Reply:
Yep, always use them. (Sorry 🙂)

Reply:
We :heart: them :wink:

Reply:
Thanks both :-1